src/govuk/helpers/_links.scss

```
@@ -369,10 +375,18 @@
   }
 }
 
-/// No underline link mixin
+/// Remove underline from links
 ///
-/// Removes underlines from links (except on hover, or when the link is active).
-/// This does not work in IE8, because IE8 does not support `:not`.
+/// Underlines are still visible when the link is hovered over, or active. This
```

```suggestion
/// Underlines are still visible when the link is hovered over or active. This
```
This is passive - can you use active voice? Probably something like: "Removes underlines from links, unless a link is active or a user is hovering over it."